modules/nf-core/toulligqc/meta.yml

```
@@ -32,6 +32,9 @@ input:
       type: file
       description: BAM file (necessary if no sequencing summary file)
       pattern: "*.bam"
+  - barcodes:
+      type: string
+      description: Coma separated barcode list (with no spaces)
```

```suggestion
      description: Comma separated barcode list (with no spaces).
      pattern: "^\\S+$",
```
if you could give an example in the description, that would be extra helpful, I think. (the pattern prohibits whitespace characters)